render_files.py

```diff
@@ -201,7 +201,7 @@ def main():
     bucket = os.environ.get('S3_BUCKET', args.s3_bucket)
     aws_access_key_id = os.environ.get("AWS_ACCESS_KEY_ID", "accessKey1")
     aws_secret_access_key = os.environ.get("AWS_SECRET_ACCESS_KEY", "verySecretKey1")
-    # openshift_release_image = os.environ.get("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE")
+    openshift_release_image = os.environ.get("OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE")
```

Comment:
and if not set?

Reply:
the job will fail on extracting openshift-installer, which is good